pkgs/applications/editors/vscode/extensions/default.nix

```diff
@@ -4080,6 +4080,23 @@ let
         };
       };
 
+      vlanguage.vscode-vlang = buildVscodeMarketplaceExtension {
+        mktplcRef = {
+          name = "vscode-vlang";
+          publisher = "vlanguage";
+          version = "0.1.14";
+          sha256 = "sha256-hlBALxBs5wZZFk4lgAkdkGs731Xuc2p0qxffOW6mMWQ=";
+        };
+        meta = {
+          description = "V language support (syntax highlighting, formatter, snippets) for Visual Studio Code.";
+          downloadPage = "https://marketplace.visualstudio.com/items?itemName=vlanguage.vscode-vlang";
+          homepage = "https://github.com/vlang/vscode-vlang";
+          license = lib.licenses.mit;
+          maintainers = [ lib.maintainers.themaxmur ];
+          platforms = [ "x86_64-linux" "aarch64-darwin" ];
```

Comment:
I just notice that you are limiting this extension to these 2 platforms only. Why ?

You did the same with your previous PR and I forgot to ask you.

Reply:
because I have two devices on these platforms, and I previously checked that this works on them, for the other two I can’t be 100% sure

Reply:
Could you provide a PR removing that attribute for your previous PR and remove this one too? The CI will test on all available platforms for you.

Reply:
Please ping me when it's done, I'll review/merge it!

Reply:
Thank you !!!